x-pack/legacy/plugins/lens/public/indexpattern_plugin/indexpattern.scss

```diff
@@ -1,5 +1,69 @@
 @import './dimension_panel/index';
 
-.lnsIndexPattern__dimensionPopover {
-  max-width: 600px;
+.lnsIndexPatternDataPanel__header {
+  display: flex;
+  align-items: center;
+  height: $euiSize * 3;
+  margin-top: -$euiSizeS;
+
+
+  & > .lnsIndexPatternDataPanel__changeLink {
+    flex: 0 0 auto;
+    margin: 0 $euiSize;
+  }
+
+  & > h4 {
```

Comment:
I'm not too worried about the exact SASS usage right now, but you might want to stay away from these generic selectors as that element could easily change. Best to stick with classes.

Reply:
Yah. Agreed. Also, the specificity of this selector is really high.